example/basic/Component.purs

```diff
@@ -40,47 +40,39 @@ component = H.parentComponent
         <> "Formless, most of which is simply Halogen boilerplate. The actual form spec and wiring "
         <> "consists of less than 20 lines of code."
     , HH.br_
-    , HH.slot unit F.component
-        { inputs
-        , validators
-        , submitter: pure <<< F.unwrapOutputFields
-        , render: renderFormless
-        }
-        (HE.input Formless)
+    , HH.slot unit F.component { initialInputs, validators, render: renderFormless } (HE.input Formless)
     ]
 
   eval :: Query ~> H.ParentDSL Unit Query ChildQuery ChildSlot Void Aff
-  eval (Formless (F.Submitted contact) a) = a <$ do
-    H.liftEffect $ log $ show (contact :: Contact)
+  eval (Formless (F.Submitted formOutputs) a) = a <$ do
+    -- To unwrap the OutputField newtypes on each field and the overall Form newtype,
+    -- use the unwrapOutputFields helper.
+    let contact :: Contact
+        contact = F.unwrapOutputFields formOutputs
+    H.liftEffect $ logShow contact
+
+  -- In this example we can ignore other outputs, but see the other examples for more
+  -- in-depth usage.
   eval (Formless _ a) = pure a
 
 -----
 -- Formless
 
-type Contact =
-  { name :: String
-  , text :: String
-  }
+type Contact = { name :: String, text :: String }
 
 newtype Form r f = Form (r
```

Comment:
Would it be clearer to call this `UserForm`? Especially now that we've taken `User` out of the Formless type signatures

Reply:
Yea, renaming to `ContactForm` is for the best. Thanks!